### ISSUE TITLE
Countries are being recorded incorrectly 

### ISSUE THREAD

**Issue by amyburness** [iwmihq/miniSASS#1102]
We also need to fix the issue with the countries where all the countries are being recorded as ZA, South Africa, or 9.

**Comment by zamuzakki** [comment]
Are we going to  use ZA, as what we have in current registration?

**Comment by zamuzakki** [comment]
I've made an action to upadate user record with country = SA, ZA, South Africa, or 9 into ZA.

**Comment by amyburness** [comment]
thanks, @zamuzakki going forward will the system only assign ZA ?

**Comment by zamuzakki** [comment]
Yes, since that's the actual value whn we select South Africa when register
